src/components/FormComponents/Autocomplete.tsx

```diff
@@ -0,0 +1,69 @@
+import { TextField, Theme } from "@mui/material";
+import AutocompleteMUI from "@mui/material/Autocomplete";
+import { Options } from "../../main/types/interfaces";
+import { makeStyles } from "@mui/styles";
+
+interface Props {
+  options: Options[];
+  value?: string;
+  onChange?: (value: string) => void;
+  inputRef?: React.Ref<HTMLInputElement>;
+  placeholder: string;
+  disabled?: boolean;
+}
+
+export const Autocomplete = ({
+  options,
+  value,
+  onChange,
+  inputRef,
+  placeholder,
+  disabled,
+}: Props) => {
+  const classes = useStyles();
+
+  const handleChange = (value: string): void => {
+    onChange?.(value || "");
+  };
+
+  const selectedOption =
+    options.find((option) => option.value === value) || null;
+
+  return (
+    <>
+      <input type="hidden" value={value || ""} ref={inputRef} />
+      <AutocompleteMUI
+        disabled={disabled}
+        value={selectedOption}
+        options={options}
+        id="grid-choose-pesticide"
+        clearOnEscape
+        renderInput={(params) => (
+          <TextField {...params} placeholder={placeholder} />
+        )}
+        classes={{ root: classes.root }}
+        onChange={(_, newValue) => {
+          handleChange(newValue?.value as string);
+        }}
+      />
+    </>
+  );
+};
```

Comment:
Let`s discuss why do you need two diff autocompletes

Reply:
Here is the main logic of Autocomplete and this component is used in AutocompleteFormField